pkg/logging/logrus.go

```diff
@@ -0,0 +1,49 @@
+package logging
+
+import (
+	"github.com/sirupsen/logrus"
+)
+
+type logrusLogger struct {
+	logger *logrus.Logger
+	fields logrus.Fields
+}
+
+func (l logrusLogger) Debug(msg string) {
+	l.logger.WithFields(l.fields).Debug(msg)
+}
+
+func (l logrusLogger) Info(msg string) {
+	l.logger.WithFields(l.fields).Info(msg)
+}
+
+func (l logrusLogger) Warn(msg string) {
+	l.logger.WithFields(l.fields).Warn(msg)
+}
+
+func (l logrusLogger) Error(msg string) {
+	l.logger.WithFields(l.fields).Error(msg)
+}
+
+func (l logrusLogger) WithField(key string, value interface{}) Logger {
+	newFields := make(logrus.Fields)
+	for k, v := range l.fields {
+		newFields[k] = v
+	}
+	newFields[key] = value
+	return logrusLogger{
+		logger: l.logger,
+		fields: newFields,
+	}
+}
+
+func getLogrusLogger() Logger {
```

Comment:
```suggestion
func newLogrusLogger() Logger {
```

It's common in Go to call factory functions for `New` or `NewSomething`.